plugins/woocommerce/src/Blocks/BlockTypes/AbstractBlock.php

```diff
@@ -437,20 +438,21 @@ protected function enqueue_data( array $attributes = [] ) {
 			$this->asset_data_registry->add(
 				'wcBlocksConfig',
 				[
-					'buildPhase'    => Package::feature()->get_flag(),
-					'pluginUrl'     => plugins_url( '/', dirname( __DIR__, 2 ) ),
-					'productCount'  => array_sum( (array) wp_count_posts( 'product' ) ),
-					'restApiRoutes' => [
+					'buildPhase'                => Package::feature()->get_flag(),
+					'experimentalBlocksEnabled' => Features::is_enabled( 'experimental-blocks' ),
```

Comment:
I think that we can avoid to add this. We can rely on the `window.wcAdminFeatures` object: WDYT?